authors/apps/authentication/tests/test_authentication.py

```diff
@@ -12,6 +12,7 @@ def test_user_registration(self):
         result = json.loads(self.new_test_user.content)
         self.assertEquals(201, self.new_test_user.status_code)
         self.assertEqual("testuser", result['user']['username'])
+        self.assertIn("you have sucessfully signedup. please check your email and verify your registration", str(result))
```

Comment:
l think this line is so long you could please consider installing flake8 to help you with such issues

Reply:
Thanks @jennizalwango . I have sorted this issue.